### DISCUSSION TITLE
Prohibiting Hot Water and Zones

### DISCUSSION THREAD

**Discussion by plejus** [gekkekoe/esphome-ecodan-hp#55] [Announcements]
I'm really interested in this hardware, but right now I have original MelCloud adapter (warranty) and I'm curious if there is possibility to prohibit hot water / zone 1 / zone 2 directly via MelCloud API? I tried to just send a message to SetAtw endpoint with EffectiveFlags = 262144 (it is described as PROHIBIT_HOT_WATER in their JS file) but it just doesn't change anything. Maybe you know  if it is possible based on your reverse engineering knowledge?

**Comment by gekkekoe** [comment]
Prohibit cmd only work in server control mode. You need to send a cmd (over cn105) to enable it, melcloud will not work.

you could run melcloud in proxy, and enable server control mode.
